server-spi-private/src/main/java/org/keycloak/models/utils/DefaultAuthenticationFlows.java

```diff
@@ -506,6 +506,15 @@ public static void clientAuthFlow(RealmModel realm) {
         execution.setAuthenticatorFlow(false);
         realm.addAuthenticatorExecution(execution);
 
+        if (Profile.isFeatureEnabled(Feature.SPIFFE_JWT)) {
```

Comment:
As an experimental feature, wouldn't it be better to force people to manually add the authenticator to their flow?

I understand this has implications for the UX, but it avoids errors when you have enabled the feature and then, after disabling it, your realm is broken because the authenticator can no longer be found.

Reply:
Not sure, there's the issue you are mentioning, but would only be if someone creates a realm after the experimental feature is enabled. I added it because it's some incredibly awkward to configure a custom auth flow.